app/build.gradle

```diff
@@ -44,6 +44,10 @@ dependencies {
     implementation 'com.karumi:dexter:5.0.0'
     implementation "com.jakewharton:butterknife:$BUTTERKNIFE_VERSION"
 
+    //paging
+    implementation "androidx.paging:paging-runtime:2.1.2"
+    implementation "androidx.paging:paging-rxjava2:2.1.2"
```

Comment:
there is a testImplementation of the pagingLibrary as well with no Android dependencies, was that not useful?

Reply:
Can you point me to the library? I didn't actually understand this comment.

Reply:
The androidx team provide a library without Android Dependencies: [paging-commons](https://developer.android.com/topic/libraries/architecture/paging#setup)

I can't find any example of why it is useful? I would have thought given the `testImplementation` that it is for writing unit tests involving PagedList etc.

Also we should be using the `-ktx` variants of these libraries, it'll give us some advantages when writing kotlin

Reply:
I have added the `-ktx` version of the library. For writing paged list tests I have added a few methods in `PagedListMock.kt` class. That seems to be serving the purpose for now. We can add the test implementation later if required.